source/get-started/cloud-install/azure.rst

```diff
@@ -1,9 +1,32 @@
 .. _azure:
 
-|CL-ATTR| on Microsoft Azure
-############################
+|CL-ATTR| on Microsoft\* Azure\*
+################################
 
-|CL-ATTR| is available for you to use in the Microsoft* Azure* marketplace and
+The Azure :abbr:`CLI (Command Line Interface)` offers the ability to create and
+manage resources in MS Azure from the command line. In this tutorial you learn
+to:
+
+#. Install the latest MS Azure CLI on your |CL| machine.
```

Comment:
Remove these steps. The contents directive creates a TOC, which presents same info.

Reply:
Done.